mcs/class/Facades/System.Threading.Overlapped/ClrThreadPoolBoundHandle.cs

```diff
@@ -148,7 +148,6 @@ public static ThreadPoolBoundHandle BindHandle(SafeHandle handle)
         /// <exception cref="ObjectDisposedException">
         ///     This method was called after the <see cref="ThreadPoolBoundHandle"/> was disposed.
         /// </exception>
-        [CLSCompliant(false)]
```

Comment:
I'd prefer turning the warning off as this file was taken verbatim from corefx and when we replace it with the corefx submodule we'll need to remember to apply this again.